travis/test_server.py

```diff
@@ -324,7 +326,6 @@ def main(argv=None):
     else:
         print("Modules to test: %s" % tested_addons)
     # setup the base module without running the tests
-    dbtemplate = "openerp_template"
```

Comment:
It's on line 285, the other one is on 286.
https://github.com/OCA/maintainer-quality-tools/pull/413/files#diff-4b42f344199e5694ffee3d2a57dd80c7R285
https://github.com/OCA/maintainer-quality-tools/pull/413/files#diff-4b42f344199e5694ffee3d2a57dd80c7R286